infra/gcp/ensure-main-project.sh

```diff
@@ -178,6 +178,24 @@ gcloud projects add-iam-policy-binding "${PROJECT}" \
     --member "serviceAccount:${MONITORING_SVCACCT_NAME}" \
     --role roles/monitoring.viewer
 
+# Kubernetes External Secrets
+color 6 -n "Ensure the kubernetes-external-secrets serviceaccount exists"
+ensure_service_account \
+    "${PROJECT}" \
+    "kubernetes-external-secrets" \
+    "Kubernetes External Secrets Service Account"
+
+color 6 -n "Empowering kubernetes-external-secrets serviceaccount to be used on aaa cluster"
+empower_ksa_to_svcacct \
+    "kubernetes-public.svc.id.goog[kubernetes-external-secrets/kubernetes-external-secrets]" \
+    "${PROJECT}" \
+    "$(svc_acct_email "${PROJECT}" "kubernetes-external-secrets")"
+
+color 6 "Empowering service account kubernetes-external-services to access payloads of the secrets"
+gcloud projects add-iam-policy-binding "${PROJECT}" \
```

Comment:
Note to self: this will need a redo or a rebase once infra script fix PR lands

Reply:
@spiffxp I switched to `ensure_project_role_binding`. Is that enough ?